modules/newRoute.js

```diff
@@ -9,7 +9,24 @@ const chalk = require('chalk'),
       routeTemplate = require('../templates/routes/route'),
       routeTest = require('../templates/routes/newRoute.test'),
       routeTestHeader = require('../templates/routes/newRouteHeader.test'),
-      routeTestEOF = require('../templates/routes/newRouteEOF.test');
+      routeTestEOF = require('../templates/routes/newRouteEOF.test'),
+
+      getFinalFileName = (fileName, route) => {
+        if (route === '/' || route.match(/^\/:/)){
+          return 'main';
+        } else if (fileName[0] === '') {
+          return fileName[1];
+        }
+        console.log(chalk.yellow('You have a route that doesn\'t begin with "/"... you should fix that'));
```

Comment:
prefer to have this wrapped in an else just for visual consistency with the rest of the if/elseif block